ERROR 2022-02-23 12:24:26,944 retry_on_timeout 152033 140460769879872 200-{"code":"100001","msg":"Trade side parameter invalid"}
Traceback (most recent call last):
  File "/home/mohammad/others/codes/intellichange/DJANGO-algorithmic_trading/global_utils/retry_on_timeout.py", line 17, in caught_func
    _result, _error = func(*args, **kwargs), False
  File "/home/mohammad/others/codes/intellichange/DJANGO-algorithmic_trading/futures_trader/exchange/exchange.py", line 49, in create_market_order
    exchange_order = self._sdk_exchange.trade_client.create_market_order(symbol=symbol,
  File "/home/mohammad/others/codes/intellichange/DJANGO-algorithmic_trading/venv/lib/python3.8/site-packages/kucoin_futures/trade/trade.py", line 370, in create_market_order
    return self._request('POST', '/api/v1/orders', params=params)
  File "/home/mohammad/others/codes/intellichange/DJANGO-algorithmic_trading/venv/lib/python3.8/site-packages/kucoin_futures/base_request/base_request.py", line 91, in _request
    return self.check_response_data(response_data)
  File "/home/mohammad/others/codes/intellichange/DJANGO-algorithmic_trading/venv/lib/python3.8/site-packages/kucoin_futures/base_request/base_request.py", line 108, in check_response_data
    raise Exception("{}-{}".format(response_data.status_code, response_data.text))
Exception: 200-{"code":"100001","msg":"Trade side parameter invalid"}
